README.md

```diff
@@ -1,89 +1,119 @@
+# get-changed-files
+
 [![CI status](https://github.com/Ana06/get-changed-files/workflows/Test/badge.svg)](https://github.com/Ana06/get-changed-files/actions?query=event%3Apush+branch%3Amain)
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](LICENSE.txt)
 
-This project is a fork of https://github.com/jitterbit/get-changed-files, which supports `pull_request_target`, allow to filter files using regular expressions and removes the ahead check.
+This project is a fork of <https://github.com/jitterbit/get-changed-files>, which supports `pull_request_target`, allow to filter files using regular expressions and removes the ahead check.
 
-# Get All Changed Files
+## Get All Changed Files
 
 Get all of the files changed/modified in a pull request (`pull_request` or `pull_request_target`) or push's commits.
 You can choose to get all changed files, only added files, only modified files, only removed files, only renamed files, or all added and modified files.
 These outputs are available via the `steps` output context.
 The `steps` output context exposes the output names `all`, `added`, `modified`, `removed`, `renamed`, and `added_modified`.
 
-# Usage
+## Usage
 
 See [action.yml](action.yml)
 
 ```yaml
-- uses: Ana06/get-changed-files@v1.2
+- uses: Ana06/get-changed-files@v2.0.0
   with:
     # Format of the steps output context.
     # Can be 'space-delimited', 'csv', or 'json'.
     # Default: 'space-delimited'
     format: ''
     # Filter files using a regex
-    filter: '*'
+    glob-filter: '*'
 ```
 
-# Scenarios
+## Scenarios
 
-- [Get all changed files as space-delimited](#get-all-changed-files-as-space-delimited)
-- [Get all added and modified files as CSV](#get-all-added-and-modified-files-as-csv)
-- [Get all removed files as JSON](#get-all-removed-files-as-json)
+- [get-changed-files](#get-changed-files)
+  - [Get All Changed Files](#get-all-changed-files)
+  - [Usage](#usage)
+  - [Scenarios](#scenarios)
+    - [Get all changed files as space-delimited](#get-all-changed-files-as-space-delimited)
+    - [Get all changed *.yml files but exclude .github/*/*.yml files](#get-all-changed-yml-files-but-exclude-githubyml-files)
+    - [Get all added and modified files as CSV](#get-all-added-and-modified-files-as-csv)
+    - [Get all removed files as JSON](#get-all-removed-files-as-json)
+  - [Install, Build, Lint, Test, and Package](#install-build-lint-test-and-package)
+  - [License](#license)
 
-## Get all changed files as space-delimited
+### Get all changed files as space-delimited
```

Comment:
I know this was already like this before, but I think it would be clearer to have this `Get all changed files as space-delimited` without the filter option and make another section to only get `*.php` and give there some more details about how the filter option works (that a list of regular expression is expected, GitHub like syntax is used, etc.). Ideally users should be able to understand how it works only with the README without checking the code. I am not completely sure this is the case now.

Reply:
Added one example